README.md

```diff
@@ -1,18 +1,41 @@
 # aaas
-Ainsley As A Service
+Ainsley As A Service. It's [placekittens](https://placekiteens.com) but for Ainsley Harriot
 
-## Requirements
+## Development
+If you want to contribute to AaaS, or self host the project, follow these steps
+### Requirements
 - PostgreSQL
 - Python
 - NodeJS
 - NVM
 
-## Setup
+### Setup
+To setup the the project locally run the following commands:
+
     ./scripts/build
+
+This command setup the virtual environment, get the required models, and create the database
```

Comment:
This doesnt actually create the database, that has to be done seperately, but I guess it could go into the script?